libs/Microsoft.MixedReality.WebRTC/Interop/InteropUtils.cs

```diff
@@ -23,6 +23,72 @@ internal struct mrsBool
         public static explicit operator bool(mrsBool b) { return (b._value != 0); }
     }
 
+    /// <summary>
+    /// Interop optional boolean, conceptually equivalent to <c>bool?</c>.
+    /// </summary>
+    [StructLayout(LayoutKind.Sequential, Size = 1)]
+    internal struct mrsOptBool
+    {
+        public static readonly mrsOptBool Unset = new mrsOptBool { _value = kUnsetValue };
+        public static readonly mrsOptBool True = new mrsOptBool(true);
+        public static readonly mrsOptBool False = new mrsOptBool(false);
+
+        private sbyte _value;
+
+        private const sbyte kUnsetValue = 0b01010101;
+
+        public bool HasValue => (_value != kUnsetValue);
+        public bool Value
+        {
+            get
+            {
+                if (!HasValue)
+                {
+                    throw new InvalidOperationException();
+                }
+                return (bool)this;
+            }
+        }
+
+        public mrsOptBool(bool value) { _value = (sbyte)(value ? -1 : 0); }
+        public mrsOptBool(bool? value)
+        {
+            if (value.HasValue)
+            {
+                _value = (sbyte)(value.Value ? -1 : 0);
+            }
+            else
+            {
+                _value = kUnsetValue;
+            }
+        }
+        public static explicit operator mrsOptBool(bool b) { return (b ? True : False); }
+        public static explicit operator mrsOptBool(bool? b)
+        {
+            if (b.HasValue)
+            {
+                return (b.Value ? True : False);
+            }
+            return Unset;
+        }
```

Comment:
What's the rationale for having both ctor and conversion operator? Can we have just the ctor (or just the operator, if we want to avoid `new` calls)?

Reply:
No rationale, I copied what was for `mrsBool`. I guess it makes sense to have only the cast operators. I removed the constructors.

Fixed.

Reply:
Actually note there's a hidden effect : the compiler will now use the implicit zeroing constructor so the default is implicitly false. With explicit constructors, well, things are more explicit 😄 

https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/using-constructors
> structs cannot contain an explicit parameterless constructor because one is provided automatically by the compiler. This constructor initializes each field in the struct to the default value.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/default-values
> Any built-in integral numeric type : 0 (zero)

Reply:
nit: I guess we could add a default ctor that initializes to unassigned, and/or making it private so it's not used externally. But not really needed.